core-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/TableSink.java

```diff
@@ -57,24 +58,29 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
     super.configurePipeline(pipelineConfigurer);
     Preconditions.checkArgument(!Strings.isNullOrEmpty(tableSinkConfig.getRowField()),
                                 "Row field must be given as a property.");
+    Schema outputSchema =
+      SchemaValidator.validateOutputSchemaAndInputSchemaIfPresent(tableSinkConfig.getSchemaStr(),
+                                                                  tableSinkConfig.getRowField(), pipelineConfigurer);
+    // NOTE: this is done only for testing
```

Comment:
can you open a jira and mention it in this comment. Jira should have something about support for passing values calculated at configure time to the runtime context.